packages/mongo/observe_multiplex.ts

```diff
@@ -140,7 +140,7 @@ export class ObserveMultiplexer {
     return !!this._isReady;
   }
 
-  _applyCallback(callbackName: string, args: any[]): void {
+  async _applyCallback(callbackName: string, args: any[]) {
     this._queue.queueTask(async () => {
```

Comment:
is really necessary turning this function to async? (this is not awaitable–as we do not [return the `scheduleRun` in AsyncQueue](https://github.com/meteor/meteor/blob/d27946d0c0bdc8a9e41dd4b37e828327d480560c/packages/meteor/async_helpers.js#L54-L61)

Reply:
No need to be async, adjusted that